src/main/java/hudson/plugins/distfork/DistForkCommand.java

```diff
@@ -269,6 +275,7 @@ private static final boolean hasBuildPermission(Collection<? extends Node> nodes
 
     /**
      * Check if the current user has permission to provision on any of the specified clouds.
+     * TODO as of 2.20 this is useless and the task will just hang anyway (see test)
```

Comment:
could perhaps just be deleted, but erring on the side of caution here

Reply:
I would keep it as is for now